nodejs/Dockerfile.tmpl

```diff
@@ -15,13 +15,7 @@ LABEL org.label-schema.schema-version="1.0" \
       org.label-schema.vcs-url=$VCS_URL \
       org.label-schema.vcs-ref=$VCS_REF \
       org.label-schema.name="nodejs" \
-      org.label-schema.description="This service provides a nodejs service platform." \
-      org.label-schema.architecture="x86_64" \
-      org.label-schema.distribution="Alpine Linux" \
-      org.label-schema.distribution-version="3.12" \
-      info.humanitarianresponse.nodejs="$NODE_VERSION" \
-      info.humanitarianresponse.npm="$NPM_VERSION" \
-      info.humanitarianresponse.yarn="$YARN_VERSION"
```

Comment:
What are we instead, then?

Reply:
There are being inherited :)
In this case, from nodejs-base
I am only re-writing the `org.label-schema.description` because is not a "base nodejs service" anymore.